packages/blocks/src/api/raw-handling/html-formatting-remover.js

```diff
@@ -42,7 +42,7 @@ export default function( node ) {
 	}
 
 	// Ignore pre content.
-	if ( node.parentElement.closest( 'pre' ) ) {
+	if ( node.parentElement.closest && node.parentElement.closest( 'pre' ) ) {
```

Comment:
What is `node.parentElement` if not an element node or `null`? Could you add a test case or leave a comment?

Reply:
@ellatrix - Do you mean a comment explaining the reason?
For what I understand, this check prevents a crash on Native Android. Would that be good for a comment?

Reply:
I would like to know the cause of the crash. What is `node.parentElement`, if not an `Element`? This line important for paste. We don't want it to be skipped, even for mobile.

https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Reply:
There is this new PR that we hope will solve the root cause:
https://github.com/wordpress-mobile/gutenberg-mobile/pull/1626

If that's the case, this PR won't be needed anymore 🙏

Reply:
Thanks!